src/main/groovy/com/bmuschko/gradle/docker/tasks/network/DockerCreateNetwork.groovy

```diff
@@ -0,0 +1,18 @@
+package com.bmuschko.gradle.docker.tasks.network
+
+import com.bmuschko.gradle.docker.tasks.AbstractDockerRemoteApiTask
+import org.gradle.api.tasks.Input
+
+class DockerCreateNetwork extends AbstractDockerRemoteApiTask {
+    @Input
+    String networkId
+
+    DockerCreateNetwork() {
+        ext.getNetworkId = { networkId }
+    }
+
+    void runRemoteCommand(final dockerClient) {
+        logger.quiet "Creating network '$networkId'."
+        dockerClient.createNetworkCmd().withName(networkId).exec()
```

Comment:
Nope https://github.com/docker-java/docker-java/blob/f6d225d2c2e65aa058da5b94c2350cbe5d5f96c9/src/main/java/com/github/dockerjava/api/command/CreateNetworkResponse.java
I'll `onNext` the response.